### PR TITLE
Add missing owners option to dotnet nugt trust repository command

### LINKED ISSUE #25724: Missing owners option for dotnet nuget trust repository command
For [dotnet nuget trust repository](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-trust#repository) sub command I forgot to include below in option part.
```
--owners <List>

Semicolon-separated list of trusted owners to further restrict the trust of a repository.
```

Below command works:
`nuget trusted-signers add .\newtonsoft.json.13.0.1-beta1.nupkg -Name NugetOrg -Repository -config ..\nuget.config -AllowUntrustedRoot -Owners "NugetTestOrg;MSTestOrg;TrustedTesOwners"`

So below one should work too:

`dotnet nuget trust repository NugetOrg ./newtonsoft.json.13.0.1-beta1.nupkg --owners "NugetTestOrg;MSTestOrg;TrustedTesOwners" --configfile ..\nuget.config --allow-untrusted-root`

### PR DESCRIPTION
**Pull request by erdembayar**
## Summary

Describe your changes here.
For [dotnet nuget trust repository](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-trust#repository) sub command I forgot to include below in option part. Here is fix for that.
```
--owners <List>

Semicolon-separated list of trusted owners to further restrict the trust of a repository.
```

Fixes: https://github.com/dotnet/docs/issues/25724

### CODE REVIEW

File: docs/core/tools/dotnet-nuget-trust.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ Adds a trusted signer with the given name, based on the repository signature or
 #### Synopsis
 
 ```dotnetcli
-dotnet nuget trust repository <NAME> <PACKAGE> [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [-v, --verbosity <LEVEL>]
+dotnet nuget trust repository <NAME> <PACKAGE> [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [--owners <List>] [-v, --verbosity <LEVEL>]
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-trust.md:142]:
```suggestion
dotnet nuget trust repository <NAME> <PACKAGE> [--allow-untrusted-root] [--configfile <PATH>] [-h|--help] [--owners <LIST>] [-v, --verbosity <LEVEL>]
```

File: docs/core/tools/dotnet-nuget-trust.md [hunk 2]
```diff
@@ -162,6 +162,10 @@ dotnet nuget trust repository <NAME> <PACKAGE> [--allow-untrusted-root] [--confi
 
 [!INCLUDE [help](../../../includes/cli-help.md)]
 
+- **`--owners <List>`**
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-trust.md:165]:
```suggestion
- **`--owners <LIST>`**
```

**Review by tdykstra — COMMENTED** [review]
LGTM, aside from a couple of minor style suggestions.

File: docs/core/tools/dotnet-nuget-trust.md (hunk 1 above)
Comment (erdembayar) [reply docs/core/tools/dotnet-nuget-trust.md]: Thank you for your review. Addressed.

File: docs/core/tools/dotnet-nuget-trust.md (hunk 2 above)
Comment (erdembayar) [reply docs/core/tools/dotnet-nuget-trust.md]: Addressed.

**Review by tdykstra — APPROVED** [review]
Thanks!

**Pull request merged**